libs/generic-view/models/src/lib/entities-delete-error.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { z } from "zod"
+
+const dataValidator = z.undefined()
+
+const configValidator = z.object({
+  modalKey: z.string(),
+  entitiesType: z.string(),
+})
+
+// export type EntitiesDeleteError = z.infer<typeof dataValidator>
```

Comment:
keep or not keep 😅 
```suggestion
export type EntitiesDeleteError = z.infer<typeof dataValidator>
```
or
```suggestion
```

Reply:
Forget to remove it 😅